src/middleware/logger.rs

```diff
@@ -100,6 +101,7 @@ impl Logger {
         Logger(Rc::new(Inner {
             format: Format::new(format),
             exclude: HashSet::new(),
+            exclude_regex: RegexSet::new(Vec::<&str>::new()).unwrap(),
```

Comment:
I got `RegexSet::empty` added to `regex` v1.4.

```suggestion
            exclude_regex: RegexSet::empty(),
```

Reply:
also will need the cargo toml regex version bumping